openapi/components/schemas/GatewayAccount.yaml

```
@@ -325,28 +326,25 @@ properties:
     type: string
     default: do-nothing
     description: |-
-      Creates zero, one, or more child transactions such as `authorize`
-      and `void`. The transactions are linked to the `setup`
-      transaction by the `parentTransactionId` relationship.
+      Creates zero, one, or more child transactions such as `authorize` and `void`.
+      The transactions are linked to the `setup` transaction by the `parentTransactionId` relationship.
     enum:
       - authorize
       - authorize-and-void
       - sca
       - do-nothing
     x-enumDescriptions:
       authorize: |-
-        Creates an `authorize` transaction in the amount/currency of
-        the request. This is used when a gateway account is configured
-        for Strong Customer Authentication (SCA).
+        Creates an `authorize` transaction in the amount and currency of the request.
+        This is used when a gateway account is configured for Strong Customer Authentication (SCA).
       authorize-and-void: |-
-        Creates an `authorize` transaction in the amount/currency of
-        the request, followed by a `void`, if the `authorize` is approved.
-        This is used when a gateway account is configured for Strong
-        Customer Authentication (SCA).
+        Creates an `authorize` transaction in the amount and currency of the request,
+        followed by a `void`, if the `authorize` is approved.
+        This is used when a gateway account is configured for Strong Customer Authentication (SCA).
       sca: |-
-        Uses Strong Customer Authentication (SCA) without an `authorize`
-        transaction. SCA includes 3dsv2, 3ds, and some specific wallet
-        behavior, such as setting up a billing agreement with PayPal.
+        Uses Strong Customer Authentication (SCA) without an `authorize` transaction.
+        SCA includes 3DS v2, 3DS, and specific wallet behavior,
```

```suggestion
        SCA includes 3DS, and specific wallet behavior,
```

I think we should remove versioning here. Because 3ds1 support is discontinue